printer/printer.lua

```diff
@@ -138,6 +138,11 @@ end
 
 
 local function set_shake(self, power, time)
+	-- Only update original position when not already shaking
+    if self.shake_time <= 0 then
```

Comment:
I think we can get rid of this get_position here? It should be ok since it already get position at init step